zzzz_modular_occulus/code/modules/core_implant/cruciform/rituals/base.dm

```diff
@@ -3,23 +3,53 @@
 //OVERRIDES: Let's just buff some of the basic litanies by reducing their cost
 
 /datum/ritual/cruciform/base/relief
-	power = 35
+	power = 25
 
 /datum/ritual/cruciform/base/soul_hunger
 	power = 35
-	desc = "Litany of piligrims, helps stave off hunger at the cost of generating considerable amounts of toxins in the user's bloodstream."
+	desc = "Litany of piligrims, helps stave off hunger by generating nutrition at the cost of generating considerable amounts of toxins in the user's bloodstream."
 
 //NEW STUFF: New shiny litanies! (haha who am i kidding its just going to be copypasta)
 
+/datum/ritual/cruciform/occulus
+	name = "occulus exclusive"
+	phrase = null
+	desc = ""
+	category = "Occulus"
```

Comment:
Tori, we can't have a tab in the ritual book that's literally Occulus. It has no meaning ICly